docs/guides/configuring-out-of-the-box-connector.md

```diff
@@ -1,6 +1,6 @@
 ---
 id: configuring-out-of-the-box-connectors
-title: Configure an out-of-the-box Connector
+title: Integrate out-of-the-box Connectors
```

Comment:
```suggestion
title: Integrate a Camunda Connector
```
Suggestion - think we might be moving to using this in place of OOTB as a term with the new Connectors work?